go.mod

```diff
@@ -1,16 +1,89 @@
 module github.com/mattermost/mattermost-plugin-gitlab
 
-go 1.16
+go 1.18
 
 require (
 	github.com/golang/mock v1.6.0
 	github.com/gorilla/mux v1.8.0
-	github.com/mattermost/mattermost-plugin-api v0.0.27
-	github.com/mattermost/mattermost-server/v6 v6.5.0
-	github.com/microcosm-cc/bluemonday v1.0.18
+	github.com/mattermost/mattermost-plugin-api v0.1.1
+	// mmgoget: github.com/mattermost/mattermost-server/v6@v7.5.0 is replaced by -> github.com/mattermost/mattermost-server/v6@21aec2741b
+	github.com/mattermost/mattermost-server/v6 v6.0.0-20221109191448-21aec2741bfe
```

Comment:
Is this intentional, i.e. downgrading from `v6.5.0` to `v6.0.0-x`. Don't have any context of this, so please excuse my ignorance

Reply:
This is not a downgrade. `github.com/mattermost/mattermost-server/v6 v6.0.0-20221109191448-21aec2741bfe` is in fact `github.com/mattermost/mattermost-server/v6@v7.5.0 `.  See https://github.com/isacikgoz/mmgoget for more context.